src/views/virtualDom/ContinuousScrollHandler.ts

```diff
@@ -207,6 +208,23 @@ export default class ContinuousScrollHandler {
   }
 
   // API
+  headElement(frameElement: HTMLElement = this.frameElement, adoElementDictionary: AdoElementDictionary = this.adoElementDictionary): AtomicDateObject | WeekDateObject {
+    // Each element adds its height until it becomes possitive.
+    let offset = frameElement.offsetTop;
+
+    const elements = Array.from(frameElement.children)
+    // teslint:disable-next-line: prefer-for-of
+    for (const element of elements) {
+      const htmlEL = element as HTMLElement;
+      const key = element.getAttribute(DATA_TAG_STRING);
+      if (offset > -(htmlEL.offsetHeight * 0.5)) {
+
+        return adoElementDictionary[key];
+      }
+
+      offset += htmlEL.offsetHeight;
+    }
```

Comment:
Confused by this. What is it meant to do? Right now if I shorten things to
```
for (const element of elements) {
    const key = element.getAttribute(DATA_TAG_STRING);
    return adoElementDictionary[key];
  }
```
I see no difference. Which makes sense as offset doesn't seem to be applied to anything...?

Reply:
the offset is how far from the top of the frame element we are. I'm checking for the intersection of the current element with the div that holds the div frame.

<img width="292" alt="Screen Shot 2020-09-28 at 12 20 09 PM" src="https://user-images.githubusercontent.com/29775131/94459348-35cf7a80-0185-11eb-8a6e-e97d5923e8bb.png">

Offset isn't applied to something. It is the distance we have traversed inside the frameElement, which lets us know if the current element is in the visible window or not.